--- config/parameters.yml.dist 2025-11-28 14:03:59.000000000 +0100
+++ config/parameters.yml.dist   2025-12-05 01:03:15.393442400 +0100
@@ -9,8 +9,8 @@
   database_host: 127.0.0.1
   database_port: ~
   database_name: claroline
-  database_user: root
-  database_password: ~
+  database_user: "%env(DB_USER)%"
+  database_password: "%env(DB_PASSWORD)%"
 
   test_database_driver: '%database_driver%'
   test_database_host: '%database_host%'

